packages/preview/enja-bib/0.1.0/src/lib.typ

```diff
@@ -0,0 +1,4 @@
+
+//add style
+#import "bib_setting_custom/bib_setting_jsme.typ"
+#import "bib_setting_custom/bib_setting_plain.typ"
```

Comment:
I think it would be better to rename these files so that they follow the kebab-case convention. Otherwise, you would have mixed casing when importing these modules.